daemon/mgr/spec_hook.go

```diff
@@ -0,0 +1,35 @@
+package mgr
+
+import (
+	"context"
+	"strings"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func setupHook(ctx context.Context, c *ContainerMeta, spec *SpecWrapper) error {
+	//if set rich mode, set initscript
+	if c.Config.Rich && c.Config.InitScript != "" {
```

Comment:
How about write this part of code like to reduce the ident and make code much more simplified:
```
if !c.Config.Rich || c.Config.InitScript == "" {
    return nil
}

args := strings.Fields(c.Config.InitScript)
if len(args) == 0{
    return nil
}
if spec.s.Hooks == nil {
   spec.s.Hooks = &specs.Hooks{} 
}
...... // blabla

```

Reply:
Good Idea!